pkg/client/restclient/transport.go

```diff
@@ -20,7 +20,7 @@ import (
 	"crypto/tls"
 	"net/http"
 
-	"k8s.io/kubernetes/pkg/client/transport"
+	"k8s.io/client-go/transport"
```

Comment:
This is a bit confusing-- the file you're editing is itself compiled into the client library. Is this really right?